CheckpointPlugin.cpp

```diff
@@ -595,14 +644,50 @@ bool CheckpointPlugin::rewind(ServerWrapper sw) {
 		return false;  // Ignored whatever inputs may have happened to exit mode; do not apply state.
 	}
 	lastRewindTime = currentTime;
-	int buttonsDown = (abs(ci.Throttle) > 0.1 ? 0x01 : 0) |
-		(abs(ci.Roll) > 0.1 ? 0x02 : 0) |
-		(ci.Handbrake ? 0x04 : 0) |
-		(ci.Jump ? 0x08 : 0) |
-		(ci.ActivateBoost ? 0x10 : 0) |
-		(ci.HoldingBoost ? 0x20 : 0) |
-		((rewindState.atCheckpoint || rewindState.justLoadedQuickCheckpoint) && abs(ci.Steer) >= .05 ? 0x40 : 0) |
-		((rewindState.atCheckpoint || rewindState.justLoadedQuickCheckpoint || abs(ci.Pitch) >= .7) && abs(ci.Pitch) >= .05 ? 0x80 : 0);
+
+	// Get user settings
+	std::string rewindInput = cvarManager->getCvar("rewind_input").getStringValue();
+	std::string matchingAxis = cvarManager->getCvar("matching_axis").getStringValue();
```

Comment:
I find the names used here confusing.  Should `rewindInput` be renamed `rewindAxis` and `activeInput` be renamed `rewindInput`?